config.toml

```diff
@@ -95,9 +95,23 @@ dev-sections = [22, 22]
 test-sections = [23, 23]
 
 [embeddings]
+algorithm = "sgns"
 prefix = "word-embeddings"
 filter = [{type = "icu-tokenizer", suppress-tags = true}, {type = "lowercase"}]
 vector-size = 50
+num-threads = 4
```

Comment:
Why should the default configuration we provide be set to use only four threads?